server/cluster/cluster.go

```diff
@@ -423,24 +423,27 @@ func (c *RaftCluster) SetStorage(s *core.Storage) {
 	c.storage = s
 }
 
-// AddSuspectRegions adds regions to suspect list.
-func (c *RaftCluster) AddSuspectRegions(ids ...uint64) {
+// AddHighPriorityScheduleRegions adds regions to be scheduled with high priority.
+// it implements HighPriority.AddHighPriorityScheduleRegions
+func (c *RaftCluster) AddHighPriorityScheduleRegions(ids ...uint64) {
 	c.Lock()
 	defer c.Unlock()
 	for _, id := range ids {
 		c.suspectRegions.Put(id, nil)
```

Comment:
Does it need to be changed?

Reply:
1. changing name here is to implement `AddHighPriorityScheduleRegions`
2. `AddHighPriorityScheduleRegions` is more clear than `AddSuspectRegions` to describe what this method actually do.

Reply:
What I mean is inside of this function, we still call `c.suspentRegions`.